docs/devices/cloud-protocol/SensorDataAndConfiguration.rst

```diff
@@ -41,6 +41,10 @@ Button is a UI element and in the case of push buttons, there is no state, which
 The messages published by the device are described in detail in the `messages JSON schema file <./messages.schema.json>`_.
 For more information, see the `messages JSON document <./message.json>`_ for an example device state.
 
+The neighboring cell measurement report is also not stored in the digital twin of the device but published via a separate topic because depending on the cloud platform its size can exceed the allowed size of the digital twin document.
```

Comment:
```suggestion
The neighboring cell measurement report is another piece of data that is not stored in the digital twin of the device.
It is published through a separate topic, because depending on the cloud platform, its size can exceed the allowed size of the digital twin document.
```